api/src/transactions.rs

```diff
@@ -1439,11 +1439,16 @@ impl TransactionsApi {
             output.gas_used(),
             exe_status,
         );
+        let mut events = output.events().to_vec();
+        let _ = self
```

Comment:
ditto

Reply:
The same answer to https://github.com/aptos-labs/aptos-core/pull/14615#discussion_r1841108898